src/main/java/gov/cms/mat/cql_elm_translation/utils/cql/CQLTools.java

```diff
@@ -226,10 +231,10 @@ private void collectUsedFunctions(
       CQLGraph graph, List<String> functions, String parentExpression) {
     for (String function : functions) {
       if (graph.isPath(parentExpression, function)) {
-        if (usedDefinitions.containsKey(function)) {
-          usedDefinitions.get(function).add(parentExpression);
+        if (usedFunctions.containsKey(function)) {
+          usedFunctions.get(function).add(parentExpression);
         } else {
-          usedDefinitions.putIfAbsent(
+          usedFunctions.putIfAbsent(
```

Comment:
I need to review this change more closely as it _may_ impact the determination of a CQL Define's Call Hierarchy. 

I think the call hierarchy processor considers CQL Functions to be CQL Defines since they are written as `define function ...`.

Reply:
@jkotanchik-SB Ok thanks. This is the one used to populate Functions section.

Reply:
Double checked, it does not impact Call Hierarchy determination. So we're good. 👍